src/MovieCards.jsx

```diff
@@ -0,0 +1,153 @@
+import { useState } from "react";
+import { useEffect } from "react";
+import './MovieCards.css'
+import { useStore } from "zustand";
+import { likedMoviesStore, searchStore } from "./App";
+
+
+export function MovieCard(props) {
+    // console.log("FROM movie card")
+    // console.log(props.data)
+    let [info, setInfo] = useState({})
+    let [liked, setLike] = useState(false)
+    
+
+    useEffect(() => {
```

Comment:
always avoid bulky useEffect, it's better to create a function let's say fetchMovieList and call that function in the useEffect

Reply:
i will give you my calo snack if you try to explore and implement react-query